v2.0/configure-replication-zones.md

```diff
@@ -306,6 +310,28 @@ Alternately, you can pass the YAML content via the standard input:
 $ echo 'num_replicas: 7' | cockroach zone set db1.t1 --insecure -f -
 ~~~
 
+### Create a Replication Zone for a Table Partition <span class="version-tag">New in v2.0</span>
+
+{{site.data.alerts.callout_info}}This is an <a href="enterprise-licensing.html">enterprise-only</a> feature.{{site.data.alerts.end}}
+
+To [control replication for table partitions](partitioning.html#replication-zones), create a YAML file defining only the values you want to change (other values will not be affected), and use the `cockroach zone set <database.table.partition> -f <file.yaml>` command with appropriate flags:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cat > australia.zone.yml
+~~~
+
+~~~ shell
```

Comment:
nit: You can remove `shell` for blocks showing a response.

Reply:
Those are not responses. They are inputs to the file.

Reply:
Ah, I see. I didn't understand what was happening with `cat >`.  We don't it this way in other zone config examples, but it's fine for now. We can revisit when we hash out more details style conventions. @lhirata, this is another topic for consideration (creating files).

Reply:
@jseldess @lhirata @rmloveland Yes, that's a new thing I found out (the hard way) recently: If you want to view a file, `cat <filename>` suffices; if you want to edit a file you need `cat > <filename>`.